data/changelog.txt

```diff
@@ -169,6 +171,14 @@ Vehicles spawn on bridges
 Added creatures to populate the new edge ocean.
 Major board game expansion
 Allow the Nunez family to move to Tacoma Commune if you help them set it up.
+Alternate Military Cargo Truck
+Added the Nag Hammadi Codices to spiritual.json
+Lockets can store photographs
+Add diet soda, because screw your free calories!
+Soldier enemies wear dogtags
+Addition of many new books, from religious texts to magazines, mainly for fluff purposes.
+Modularize Desert Eagle, add .50 AE
+Adds the Steyr Scout rifle
```

Comment:
```suggestion
Lockets can store photographs
Add diet soda, because screw your free calories!
Addition of many new books, from religious texts to magazines, mainly for fluff purposes.
Modularize Desert Eagle, add .50 AE
```